generators/server/templates/quarkus/src/main/resources/application.properties.ejs

```diff
@@ -74,3 +76,18 @@ quarkus.container-image.group=
 quarkus.container-image.name=<%= baseName %>
 quarkus.container-image.tag=latest
 <%_ } _%>
+
+<%if (authenticationType === 'oauth2') { _%>
+# Oauth 2.0 and OIDC
+quarkus.oidc.enabled=true
+quarkus.oidc.auth-server-url=http://localhost:9080/auth/realms/jhipster/
+quarkus.oidc.client-id=web_app
+quarkus.oidc.credentials.secret=web_app
+quarkus.oidc.application-type=hybrid
+quarkus.oidc.authentication.scopes=profile,address,email,jhipster,microprofile-jwt,offline_access,phone
```

Comment:
We should only use standard OIDC scopes here so users don't have to set up new scopes on their identity provider. https://openid.net/specs/openid-connect-core-1_0.html#ScopeClaims

In other words, remove `jhipster` and `microprofile-jwt` (unless they're necessary for some reason).